lib/octocatalog-diff/catalog-util/builddir.rb

```diff
@@ -330,7 +334,7 @@ def install_ssl_client_password(logger, password)
       end
 
       def environment
-        @options[:preserve_environments] ? @options.fetch(:environment, 'production') : 'production'
+        @options.fetch(:environment, 'production')
```

Comment:
Maybe there should be a protection here to block `environment` and `create-symlinks` being used together, as the debug output on 89 states.